.gitignore

```diff
@@ -14,6 +14,7 @@ config/honeybadger.yml
 config/cli_environments/stage.rb
 config/cli_environments/local.rb
 db/*.sqlite3
+db/schema.rb
```

Comment:
Why are we putting this in gitignore? Typically this should be checked in. 
see other projects:
https://github.com/sul-dlss/argo/blob/master/db/schema.rb
https://github.com/sul-dlss/SearchWorks/blob/master/db/schema.rb

Reply:
whoops.  will fix - one sec.